docs/verify/index.mdx

```diff
@@ -4,9 +4,18 @@ description: Introduction to the Verify phase
 ---
 # Introduction
 
-In the verify phase, you test your quantum programs by running them on simulated devices and exploring their performance under realistic device [noise models](https://qiskit.org/ecosystem/aer/apidocs/aer_noise.html). This allows you to validate them before sending them to a physical system. This phase consists of the following steps:
+In the verify phase, you test your quantum programs by running them on simulated devices and exploring their performance under realistic device noise models. This allows you to validate them before sending them to a physical system.
 
-1. Run your program on a quantum simulator, using [Qiskit reference primitives](simulate-with-qiskit-primitives) or [IBM Quantum cloud-based simulators](cloud-based-simulators).
-1. [Add noise to your simulation](using-ibm-quantum-simulators).
-2. Optionally [create your own noise model](building_noise_models). 
+Because the cost of classically simulating quantum circuits scales exponentially with the number of qubits, circuits that are larger than 50 qubits or so generally cannot be directly verified. For such circuits, you can:
 
+- Test smaller versions of the circuits that can be simulated classically.
+- Modify the circuits so that they become classically simulable and test these modified circuits.
+
+Stabilizer circuits are a useful tool for accomplishing this latter goal. See [Efficient simulation of stabilizer circuits with Qiskit Aer primitives](stabilizer-circuit-simulation) for an overview of stabilizer circuits and how to simulate them efficiently.
+
+The following tools are available for you to verify your quantum programs:
+
+- For exact simulation of small quantum circuits, you can use the reference primitives included with Qiskit. See [Exact simulation with Qiskit primitives](simulate-with-qiskit-primitives).
+- For higher-performance simulation that can handle larger circuits, or to incorporate noise models into your simulation, use [Qiskit Aer](https://qiskit.org/ecosystem/aer/), a project that is part of the [Qiskit Ecosystem](https://qiskit.github.io/ecosystem/). See [Exact and noisy simulation with Qiskit Aer primitives](simulate-with-qiskit-aer).
+- To build your own custom noise models, use the [`noise`](https://qiskit.org/ecosystem/aer/apidocs/aer_noise.html) module of Qiskit Aer. See [Building noise models](building_noise_models).
+- As mentioned before, Qiskit Aer can be used to efficiently simulate stabilizer circuits, a special restricted class of quantum circuits. For example, using Qiskit Aer, you can simulate stabilizer circuits with thousands of qubits. See [Efficient simulation of stabilizer circuits with Qiskit Aer primitives](stabilizer-circuit-simulation).
```

Comment:
feels a bit repetitive to have this bullet point again about extended stabilisers. Maybe instead adjust the sentence before the bullet points to say "in addition, the following tools are available..."

Reply:
Done.